### DISCUSSION TITLE
Supporter Case

### DISCUSSION THREAD

**Discussion by D3VL-Clint** [alf45tar/PedalinoMini#294] [General]
Hello, I supported the project, but unable to download STL files for 3d print?
just checking I havent missed anything.

Thanks! :)
Clint

**Comment by alf45tar** [comment]
Just added to private sponsor repository. Try the link now.

**Reply by D3VL-Clint** [reply]
THANK YOU!!!
and amazing work on this project!

**Comment by alf45tar** [comment]
For your information I am working on one PCB design for led strip and one for the foot switches. It will simplify a lot the wiring connections and the final assembly of PedalinoMini Case 1.

**Reply by D3VL-Clint** [reply]
is there an ETA? :) or just here when its here?

**Reply by alf45tar** [reply]
There is not ETA for the moment. There are some changes under review to find the best design in a so compact case.
